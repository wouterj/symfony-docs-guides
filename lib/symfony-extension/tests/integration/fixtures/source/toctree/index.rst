Toctree
=======

Simple

.. toctree::

    file


Simple, titles only

.. toctree::
    :titlesonly:

    file

Simple, file from other directory

.. toctree::

    directory/another_file

Glob

.. toctree::
    :glob:

    *

Glob, with explicit order

.. toctree::
    :glob:

    file1
    *

Hidden

.. toctree::
    :hidden:

    file
