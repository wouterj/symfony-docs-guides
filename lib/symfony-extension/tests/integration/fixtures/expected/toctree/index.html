<!DOCTYPE html>
<html>
    <head>
        <meta charset="utf-8" />
    </head>
    <body>
        <div class="section" id="toctree">
            <h1><a class="headerlink" href="#toctree" title="Permalink to this headline">Toctree</a></h1>
            <p>Simple</p>
            <div class="toctree-wrapper toc-size-md">
                <ul class="toctree toctree-level-1 toctree-length-1">
                    <li>
                        <a href="/file.html">Title</a>
                        <ul class="toctree toctree-level-2 toctree-length-1">
                            <li>
                                <a href="/file.html#sub-title">Sub title</a>
                            </li>
                        </ul>
                    </li>
                </ul>
            </div>
            <p>Simple, titles only</p>
            <div class="toctree-wrapper toc-size-md">
                <ul class="toctree toctree-level-1 toctree-length-1">
                    <li>
                        <a href="/file.html">Title</a>
                    </li>
                </ul>
            </div>
            <p>Simple, file from other directory</p>
            <div class="toctree-wrapper toc-size-md">
                <ul class="toctree toctree-level-1 toctree-length-1">
                    <li>
                        <a href="/directory/another_file.html">Another file</a>
                    </li>
                </ul>
            </div>
            <p>Glob</p>
            <div class="toctree-wrapper toc-size-md">
                <ul class="toctree toctree-level-1 toctree-length-2">
                    <li>
                        <a href="/file.html">Title</a>
                        <ul class="toctree toctree-level-2 toctree-length-1">
                            <li>
                                <a href="/file.html#sub-title">Sub title</a>
                            </li>
                        </ul>
                    </li>
                    <li>
                        <a href="/file1.html">Title 1</a>
                    </li>
                </ul>
            </div>
            <p>Glob, with explicit order</p>
            <div class="toctree-wrapper toc-size-md">
                <ul class="toctree toctree-level-1 toctree-length-2">
                    <li>
                        <a href="/file1.html">Title 1</a>
                    </li>
                    <li>
                        <a href="/file.html">Title</a>
                        <ul class="toctree toctree-level-2 toctree-length-1">
                            <li>
                                <a href="/file.html#sub-title">Sub title</a>
                            </li>
                        </ul>
                    </li>
                </ul>
            </div>
            <p>Hidden</p>
        </div>
    </body>
</html>
